WcaOnRails/app/models/regulation.rb

```diff
@@ -0,0 +1,21 @@
+class Regulation < SimpleDelegator
+  begin
+    REGULATIONS = JSON.parse(File.read(Rails.root.to_s + "/app/views/regulations/wca-regulations.json")).freeze
+  rescue
+    REGULATIONS = [].freeze
```

Comment:
I see two options:
 - make travis build the json
 - restore the `begin...rescue`

Do you have a strong preference?